app/javascript/api/adapters/listing/RentalListingAdapter.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Listing,
+  ListingStatus,
+  CSVFormattingType,
+  CountyCode,
+} from "@bloom-housing/backend-core/types"
+
+import RailsRentalListing from "../../types/rails/listings/RailsRentalListing"
+import { Adapter } from "../adapter"
+import ListingPropertyAdapter from "./ListingPropertyAdapter"
+
+const toDate = (s: string): Date => new Date(s) // todo: handle different formats, actually implement this properly
+
+const ListingAdapter: Adapter<RailsRentalListing, Listing> = (item: RailsRentalListing) => ({
+  status: ListingStatus.active,
```

Comment:
hm, is it possible to not default to a status of active here in this PR? or does that need to be in a follow-up?